packages/components/src/components/hds/form/key-value-pair/yield.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="hds-key-value-pair__generic-container" {{this._onInsert}}>
+  <span />
```

Comment:
Question: I added this span to force the yielded content into the 2nd row with the inputs + error text. I am not sure of a better way to do this.